Document the storeFormat option for create database (#1577)
neo4j/docs-operations: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

----

In Neo4j, the default store format for all new databases is `aligned`.
If you want to change it, you can set a new value for the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] configuration in the _neo4j.conf_ file. +

> Alternatively, you can set the store format of a new database using the `CREATE DATABASE databasename OPTIONS {storeFormat: 'the-new-format'}` command. +

Author: ```suggestion
Alternatively, you can set the store format of new databases using the `CREATE DATABASE databasename OPTIONS {storeFormat: 'the-new-format'}` command. 
However, if the store is seeded with `seedURI` or `existingDataSeedInstance`, or if the command is being used to mount pre-existing store files already present on the disk, they will use their current store format without any alterations.
```

| `use`
|
Controls how the system handles existing data on disk when creating the database.

> Currently, this is only supported with `existingDataSeedInstance` and must be set to `use` which indicates the existing data files should be used for the new database.

Reviewer: There is also `seedURI` now isn't there? I'm not sure if that requires `existingData` or not though. I think it probably should.

> Currently, this is only supported with `existingDataSeedInstance` and must be set to `use` which indicates the existing data files should be used for the new database.

Author: ```suggestion
Currently, this is only supported with `existingDataSeedInstance` and `seedURI`, and must be set to `use` which indicates the existing data files should be used for the new database.
```

| `txLogEnrichment`
| `FULL` \| `DIFF` \| `NONE`
|
Defines the level of enrichment applied to transaction logs for Change Data Capture (CDC) purposes.

For details about enrichment mode, see link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#set-enrichment-mode/[Change Data Capture Manual -> Enable CDC on self-managed instances -> Set the enrichment mode].

| `storeFormat`
| `aligned` \| `standard` \| `high_limit` \| `block`
|

> Defines the store format of a new database.

Reviewer: We should also state that this is only effective if the database is created new. If the store is seeded with `seedURI`, or `existingDataSeedInstance` or if the command is being used to mount existing store files already on the disk they will use their current store format - there will not be any migration.

> Defines the store format of a new database.

Reviewer 2: ```suggestion
Defines the store format if the database created is new.
If the store is seeded with `seedURI` or `existingDataSeedInstance`, or if the command is being used to mount pre-existing store files already present on the disk they will use their current store format without any alterations.
```
